Lib/dis.py

```diff
@@ -283,7 +290,7 @@ def _disassemble(self, lineno_width=3, mark_as_current=False, offset_width=4):
         if self.arg is not None:
             fields.append(repr(self.arg).rjust(_OPARG_WIDTH))
             # Column: Opcode argument details
-            if self.argrepr:
+            if self.argrepr and self.argval is not UNKNOWN:
```

Comment:
This suggests that if argval is UNKNOWN, we could set argrepr to None or an empty string, rather than testing argval?

Reply:
We could set it to empty string. I didn't want to make it None because now it's always a string.